bin/main.rb

```diff
@@ -1,2 +1,54 @@
 #!/usr/bin/env ruby
-puts 'Hello World'
+
+puts 'Welcome to Tic Tac Toe!'
+print 'Enter player 1 name: '
+player_one = gets.chomp
+
+print 'Good. Enter player 2 name: '
+player_two = gets.chomp
+
+puts "\nPlayer 1 is #{player_one} using symbol X. \nPlayer 2 is #{player_two} using symbol O. \n\nLet's Play!"
+
+# A game board will be displayed to the user, consisting of numbers from 1 to 9
+
+# We also have arrays that stores the values of selected moves by the players
+p1_move = []
+p2_move = []
+
+puts "\nWith each boxes displayed from 1 to 9, each player can type in a value >=1 and <= 9"
+
+count = 0
+while count < 9
```

Comment:
[Suggestion for your next milestone]
Your loop should not depend on any constant number. Cause it reduces the scalability of your code.

Reply:
This will be taken care of in the next milestone.